src/encoded/static/components/app.js

```diff
@@ -90,6 +90,8 @@ const biosampleTypeList = [
     'stem cell',
     'in vitro differentiated cells',
     'induced pluripotent stem cell line',
+    "single cell",
+    "cell-free sample"
```

Comment:
To fit AirBnB style guides, this last item should have a trailing comma. I have changes to app.js coming up, so I’d recommend not worrying about this here and just fixing in my branch since this will show up in red once I rebase.

Reply:
My ENCD-3938 PR will conflict directly with the app.js changes here, so the trailing comma will definitely get fixed once I resolve this conflict.